backend/migrations/versions/6c9a8afee4e4__alembic_sync.py

```diff
@@ -17,147 +19,169 @@
 
 
 def upgrade():
+    envname = os.getenv('envname', 'local')
```

Comment:
By default we use `dckrcompose`, don't we?

Reply:
For AWS deployment - `envname` will be the `envname` of the particular deployment account

For local deployment with docker - `envname` will be `dkrcompose`

If not set thought best to default to `local` - as we do with other migration scripts such as:
-  `dataall/backend/migrations/versions/4f3c1d84a628_modify_notifications_column_types.py`
- `dataall/backend/migrations/versions/a1049d854e29_implement_dataset_locking_mechanism.py`
etc.